python/tk_desktop/desktop_engine_site_implementation.py

```diff
@@ -355,25 +355,6 @@ def get_current_login(self):
         """
         return self._current_login
 
-    def check_login_based(self, core_path):
```

Comment:
This whole need to check if something is login based is a very complicated means to simply refresh the credentials if the user is login based.